src/css/index.scss

```diff
@@ -0,0 +1,103 @@
+.Info-text1{ 
+    width: 660px;
+    height: 630px;
+    background-image: repeating-linear-gradient(120deg,#3498bd,#8e44ad); 
+    color: white;
+    font-family: sans-serif;
+   }
+   
+   .h2-qu1{
+      padding: 40px 40px;
+      font-size: 40px;
+   }
+   .h2-qu2{
+      padding: 40px 40px;
+      font-size: 40px;
+      color: black;
+      font-family: sans-serif;
```

Comment:
you are repeat yourself, you need a class to extra all the common styles and apply to all the html tags that needs it.